<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meat</class>
 <widget class="QDialog" name="meat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мясо</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Бедро куриное НПФ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Филе грудки индейки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Филе грудки индейки ИНДИ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Голень куриная АБ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Тушка цыпленка АБ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Стейк из грудки индейки ИНДИ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Голень куриная с кожей</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Карбонад из свинины НЕЛЕНТА</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Окорок свиной НЕЛЕНТА</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Бекон ЧЕРКИЗОВО</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Говядина МИРАТОРГ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Фарш из курицы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Фарш из индейки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Фарш говяжий</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Шея свиная</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
